fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/models/FedoraResource.java

```diff
@@ -257,4 +257,10 @@ void replaceProperties(final IdentifierConverter<Resource, FedoraResource> idTra
      * @return the resource being described
      */
     FedoraResource getDescribedResource();
+
+    /**
+     * Get a graph resource corresponding to this resource
+     * @return a graph resource
+     */
+    Resource graphResource(Converter<Resource, String> idTranslator);
```

Comment:
I think I have addressed all of these that I can in the last commit; in the
next I will delete IdentifierConverter and hopefully we can get agreement
on a squashable set of changes.

On Thu, Aug 4, 2016 at 4:36 PM, A. Soroka <notifications@github.com> wrote:

> In fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/
> models/FedoraResource.java
> <https://github.com/fcrepo4/fcrepo4/pull/1086#discussion_r73597875>:
>
> > @@ -257,4 +257,10 @@ void replaceProperties(final IdentifierConverter<Resource, FedoraResource> idTra
> >       * @return the resource being described
> >       */
> >      FedoraResource getDescribedResource();
> > +
> > +    /**
> > +     * Get a graph resource corresponding to this resource
> > +     * @return a graph resource
> > +     */
> > +    Resource graphResource(Converter<Resource, String> idTranslator);
>
> I'll suggest going ahead and calling this a URI. It's going to be a URI,
> and that's what people want.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/fcrepo4/fcrepo4/pull/1086/files/f4c90a8742a4337d6df3f673276e13abd4da98c4#r73597875>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAHUNEm8ALgYYcqDswNsGTi_Ui7H0T7kks5qck1VgaJpZM4JdBw8>
> .
>